<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="42612" started-at="2020-04-13T10:26:34 EDT" finished-at="2020-04-13T10:27:17 EDT">
    <groups>
    </groups>
    <test name="Default test" duration-ms="42612" started-at="2020-04-13T10:26:34 EDT" finished-at="2020-04-13T10:27:17 EDT">
      <class name="com.etsy.Tests.RegisterTest">
        <test-method status="PASS" signature="startExtent()[pri:0, instance:com.etsy.Tests.RegisterTest@515aebb0]" name="startExtent" is-config="true" duration-ms="136" started-at="2020-04-13T10:26:34 EDT" finished-at="2020-04-13T10:26:34 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startExtent -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.etsy.Tests.RegisterTest@515aebb0]" name="setup" is-config="true" duration-ms="9723" started-at="2020-04-13T10:26:34 EDT" finished-at="2020-04-13T10:26:44 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="verifyHeadingOnRegisterPage(java.lang.reflect.Method)[pri:1, instance:com.etsy.Tests.RegisterTest@515aebb0]" name="verifyHeadingOnRegisterPage" duration-ms="30888" started-at="2020-04-13T10:26:44 EDT" description="Verify that the heading on the Register page is Create your account and assert that the actual heading andand expected heading are same" finished-at="2020-04-13T10:27:15 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.etsy.Tests.RegisterTest.verifyHeadingOnRegisterPage(java.lang.reflect.Method)]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Please check your actual and expected text --  expected [Create your account] but found [ ]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Please check your actual and expected text --  expected [Create your account] but found [ ]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at com.etsy.Tests.RegisterTest.verifyHeadingOnRegisterPage(RegisterTest.java:43)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyHeadingOnRegisterPage -->
        <test-method status="PASS" signature="teardown(org.testng.ITestResult)[pri:0, instance:com.etsy.Tests.RegisterTest@515aebb0]" name="teardown" is-config="true" duration-ms="1391" started-at="2020-04-13T10:27:15 EDT" finished-at="2020-04-13T10:27:16 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyHeadingOnRegisterPage status=FAILURE method=RegisterTest.verifyHeadingOnRegisterPage(java.lang.reflect.Method)[pri:1, instance:com.etsy.Tests.RegisterTest@515aebb0] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
        <test-method status="PASS" signature="endReport()[pri:0, instance:com.etsy.Tests.RegisterTest@515aebb0]" name="endReport" is-config="true" duration-ms="418" started-at="2020-04-13T10:27:16 EDT" finished-at="2020-04-13T10:27:17 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- com.etsy.Tests.RegisterTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
